core/templates/components/summary-tile/learner-topic-goals-summary-tile.component.ts

```diff
@@ -104,17 +106,19 @@ export class LearnerTopicGoalsSummaryTileComponent implements OnInit {
     if (this.incompleteStoryNodes.length > 0) {
       this.storyNodeToDisplay = this.incompleteStoryNodes[0];
 
-      if (this.storyNodeToDisplay.getThumbnailFilename()) {
+      let thumbnailFilename = this.storyNodeToDisplay.getThumbnailFilename();
+      if (thumbnailFilename) {
```

Comment:
what else are we not expecting our thumnailFilename to be? What should happen if the property doesn't exist and is null or undefined?

Reply:
Done